src/commands/RedactCommand.ts

```diff
@@ -18,7 +18,7 @@ import { Mjolnir } from "../Mjolnir";
 import { redactUserMessagesIn } from "../utils";
 import { Permalinks } from "@vector-im/matrix-bot-sdk";
 
-// !mjolnir redact <user ID> [room alias] [limit]
+// !mjolnir redact <user ID> [room alias] [limit] --quarantine
 export async function execRedactCommand(roomId: string, event: any, mjolnir: Mjolnir, parts: string[]) {
```

Comment:
same as quarantine command I think it makes sense to check if bot can execute quarantine call and throw an error if not if the flag provided

Reply:
Done.